### PR TITLE
Enh/more methods

### PR DESCRIPTION
**Pull request by Joel-Dahne**
I have added functions from `acb_hypgeom.h` and `arb_hypgeom.h`. They both parsed cleanly, only 6 (of 212) functions failed to parse due to types we have not implemented (`acb_poly_struct *` and `fmpz_t`).

The files that are left that we are not parsing are
- `arb_fmpz_poly.h`
- `acb_dft.h`
- `acb_elliptic.h`
- `acb_modular.h`
- `dirichlet.h`
- `acb_dirichlet.h`
- `bernoulli.h`
- `hypgeom.h`
- `partitions.h`
- `arb_calc.h`
- `acb_calc.h`
- `fmpz_extras.h`
- `bool_mat.h`
- `dlog.h`
- `fmpr.h`

However most of these are not relevant at all (for example`bool_mat.h`) whereas others will required a mostly manual implementation (`arb_calc.h`).

### CODE REVIEW

**Comment by kalmarek** [comment]
`acb_calc.h` should be covered by #66; other methods in the header are legacy ones

**Comment by Joel-Dahne** [comment]
Should we merge this now or should I add parsing of some more header files first?

**Comment by kalmarek** [comment]
since you're on `parse.jl` could you maybe add the two-letters codes suggested somewhere? like
`#ut arb_signature_here` for unsupported type, `#pf ...` for parse failure and maybe `#mo ...` for manual override?

It'd be great if each line with `#mo ...` could also get a short explanation why manual override is done (such information is forgotten in a week after merge ;) 

otherwise, I'm happy with merging

File: parse/parse.jl [hunk 1]
```diff
@@ -73,14 +81,26 @@ function generate_file(title, sections)
                 str *= "arbcall\"" * s * "\"\n"
             catch e
                 if e isa Arblib.UnsupportedArgumentType
+                    push!(unsuppargs, e.key)
+                    num_unsupparg += 1
                     str *= "# arbcall\"" * s * "\"\n"
```
Comment (kalmarek) [inline parse/parse.jl:86]:
```suggestion
                    str *= "#ut arbcall\"" * s * "\"\n"
```

File: parse/parse.jl [hunk 2]
```diff
@@ -73,14 +81,26 @@ function generate_file(title, sections)
...
                 elseif e isa KeyError
+                    push!(keyerrors, e.key)
+                    num_keyerror += 1
                     str *= "# arbcall\"" * s * "\"\n"
```
Comment (kalmarek) [inline parse/parse.jl:90]:
```suggestion
                    str *= "#pf arbcall\"" * s * "\"\n"
```

**Comment by Joel-Dahne** [comment]
Good point! I'll take care of that!

**Comment by Joel-Dahne** [comment]
How does this look?

**Review by kalmarek — APPROVED** [review]
wow, you went full throttle :1st_place_medal: !

**Pull request merged**